src/optimum/rbln/configuration_utils.py

```diff
@@ -774,10 +794,17 @@ def _prepare_for_serialization(self) -> Dict[str, Any]:
             if isinstance(value, RBLNSerializableConfigProtocol):
                 # Convert nested RBLNModelConfig to its serializable form
                 serializable_map[key] = value._prepare_for_serialization()
+            elif key == "_dtype":
+                serializable_map["dtype"] = (
+                    RBLNCompileConfig.normalize_dtype(value) if isinstance(value, torch.dtype) else value
+                )
```

Comment:
isn't `_dtype` always `str` type?
```suggestion
            elif key == "_dtype":
                serializable_map["dtype"] = value
```

Reply:
Yes. If there were a guarantee that the dtype would always remain a string, simple assignment would work. However, I implemented it this way because performing a defensive normalize seems a bit safer.

Reply:
Would you prefer to remove it?

Reply:
I think it's over-engineering.